static/elements/chromedash-guide-verify-accuracy-page.js

```diff
@@ -51,6 +56,11 @@ export class ChromedashGuideVerifyAccuracyPage extends LitElement {
     });
   }
 
+  disconnectedCallback() {
```

Comment:
I feel that this is too repetitive.
How about defining a base class for all our page classes?

Reply:
Yeah, I think it can be good, especially the form handling methods are shared among multiple pages too. I'll probably do it a follow-up PR.

Also, just FYI, I also deleted pages/featuredetails.py and pages/featuredetails_test.py because these are no longer used too and the unit test was complaining there's no feature.html.